derive_builder_core/src/builder.rs

```diff
@@ -84,6 +89,24 @@ impl<'a> ToTokens for Builder<'a> {
                 .unwrap_or((None, None, None));
             let builder_fields = &self.fields;
             let functions = &self.functions;
+
+
+            // Create the comma-separated set of derived traits for the builder
+            let derived_traits = {
+                let default_trait: Ident = parse_quote!(Default);
+                let clone_trait: Ident = parse_quote!(Clone);
+
+                let mut traits: Punctuated<&Ident, Token![,]> = Default::default();
+                traits.push(&default_trait);
+
+                if self.must_derive_clone {
```

Comment:
Should this also check if the builder pattern is non-owned?
The tests use the `default_builder!` macro, which has `must_derive_clone = true`, but `Options#as_builder` only checks if any fields require clone. I've tried tracing the code but haven't found the answer

Reply:
This shouldn't, but I was wondering if the function in `darling_opts` should.